Error Building on Microsoft Container mcr.microsoft.com/dotnet/core/sdk

This line from build.sh returns the value of the net framework twice.

local targetFramework=$(awk -F: '/<TargetFramework(s)?>.*netcoreapp[1-9]\.[0-9].*<\/TargetFramework(s)?>/ { print $0 }' $file | sed 's/.*\(netcoreapp[1-9]\.[0-9]\).*/\1/')

Tested adding ' | tail -1' and it builds successfully.

Reproduction steps:
````
docker run -it mcr.microsoft.com/dotnet/core/sdk:2.1 bash
git clone https://github.com/microsoft/dotnet-apiport.git
cd dotnet-apiport
./build.sh
````
Get the output (Notice duplication: "Building for: **netcoreapp2.1 netcoreapp2.1**")

Building ApiPort... Configuration: [Debug]
Building for: netcoreapp2.1 netcoreapp2.1
Microsoft (R) Build Engine version 16.0.450+ga8dc7f1d34 for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

MSBUILD : error MSB1008: Only one project can be specified.
Switch: netcoreapp2.1

For switch syntax, type "MSBuild -help"

--- a/build.sh
+++ b/build.sh
@@ -64,7 +64,7 @@ installSDK() {
 
 tryGetTargetFramework() {
 	local file=$1
-	local targetFramework=$(awk -F: '/<TargetFramework(s)?>.*netcoreapp[1-9]\.[0-9].*<\/TargetFramework(s)?>/ { print $0 }' $file | sed 's/.*\(netcoreapp[1-9]\.[0-9]\).*/\1/')
+	local targetFramework=$(awk -F: '/<TargetFramework(s)?>.*netcoreapp[1-9]\.[0-9].*<\/TargetFramework(s)?>/ { print $0 }' $file | sed 's/.*\(netcoreapp[1-9]\.[0-9]\).*/\1/' | tail -1)
 	echo $targetFramework
 }
 

--- a/build.sh
+++ b/build.sh
@@ -64,7 +64,7 @@ installSDK() {
 
 tryGetTargetFramework() {
 	local file=$1
-	local targetFramework=$(awk -F: '/<TargetFramework(s)?>.*netcoreapp[1-9]\.[0-9].*<\/TargetFramework(s)?>/ { print $0 }' $file | sed 's/.*\(netcoreapp[1-9]\.[0-9]\).*/\1/' | tail -1)
+	local targetFramework=$(awk -F: '/<TargetFramework(s)?>.*netcoreapp[1-9]\.[0-9].*<\/TargetFramework(s)?>/ { print $0 }' $file | sed 's/.*\(netcoreapp[1-9]\.[0-9]\).*/\1/' | tail -n 1)
 	echo $targetFramework
 }
 

